evennia/contrib/utils/debugpy/README.md

```diff
@@ -0,0 +1,157 @@
+# In-game debugpy command
+
+This registers an in-game command `debugpy` which starts the debugpy debugger and listens on port 5678.
+For now this is only available for Visual Studio Code (VS Code).
+
+If you are a JetBrains PyCharm user and would like to use this, make some noise at:
+https://youtrack.jetbrains.com/issue/PY-63403/Support-debugpy
+
+
+Credit for this goes to Moony on the Evennia Discord getting-help channel, thx Moony!
+
+0xDEADFED5 simply tied a pretty bow around it and stuck it here for everybody else.
+
+
+
+## Dependencies
+
+This requires VS Code and debugpy, so make sure you're using VS Code.
+
+From the venv where you installed Evennia run:
+
+`pip install debugpy`
+
+## Enable the command in Evennia
+
+In your Evennia mygame folder, open up `/commands/default_cmdsets.py`
+
+add `from evennia.contrib.utils.debugpy.cmd import CmdDebugPy` somewhere near the top.
+
+in CharacterCmdSet.at_cmdset_creation add this under `super().at_cmdset_creation()`:
+
+`self.add(CmdDebugPy)`
+
+For a newly initialized game, the result would look like this at the top of the file:
+
+```python
+"""
+Command sets
+
+All commands in the game must be grouped in a cmdset.  A given command
+can be part of any number of cmdsets and cmdsets can be added/removed
+and merged onto entities at runtime.
+
+To create new commands to populate the cmdset, see
+`commands/command.py`.
+
+This module wraps the default command sets of Evennia; overloads them
+to add/remove commands from the default lineup. You can create your
+own cmdsets by inheriting from them or directly from `evennia.CmdSet`.
+
+"""
```

Comment:
I would suggest commenting this out to make this shorter. Copying the entire module also makes this less future-proof if the original changes.